monolith/services/restaurant_services.py

```diff
@@ -111,3 +111,23 @@ def get_reservation_rest(owner_id, restaurant_id, from_date, to_date, email):
         # execute and retrive results...
         result = db.engine.execute(stmt, params)
         return result.fetchall()
+
+    @staticmethod
+    def reviewRestaurant(restaurant_id, reviewer_id, stars, review):
+        """
+        This method insert a review to the specified restaurant
+        """
+        print("check")
+        if stars < 0 or stars > 5:
+            return None
+
+        new_review = Review()
+        new_review.restaurant_id = restaurant_id
+        new_review.reviewer_id = reviewer_id
+        new_review.stars = stars
+        new_review.review = review
+
+        db.session.add(new_review)
+        db.session.commit()
+
+        return True
```

Comment:
Can you test this code before joining with the main? I can test it if you have other work to do

Reply:
yes, I have to add the tests!

Reply:
thanks, i will make the form test